packages/main/src/Tree.hbs

```diff
@@ -9,50 +9,14 @@
     @ui5-item-click="{{_onListItemClick}}"
     @ui5-item-delete="{{_onListItemDelete}}"
     @ui5-selection-change="{{_onListSelectionChange}}"
+    @ui5-toggle="{{_onListItemToggle}}"
+    @ui5-step-in="{{_onListItemStepIn}}"
+    @ui5-step-out="{{_onListItemStepOut}}"
+    @mouseover="{{../_onListItemMouseOver}}"
```

Comment:
no need for `../` here and on the next line, it works by coincidence, but would not be guaranteed to work in the future if there is no upper scope (same for next line)